src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
I'm pushing for these names in rippled:
`AccountID` - 160 bit account identifier
`AccountPublicKey` - Public key for an account
`PublicKey` - A general public key